security_account.tf

```diff
@@ -48,6 +48,7 @@ module "security_account" {
 
 # And delegate power to it
 resource "aws_organizations_delegated_administrator" "cloudformation" {
+  count             = local.security_services["disable_stacksets"] ? 0 : 1
```

Comment:
Should this be `local` or `var`?

Reply:
There's a local map merge happening [here](https://github.com/primeharbor/org-kickstart/blob/main/main.tf#L32) so it's used. However with the variable type being set properly it's possible to just remove this entirely.